packages/input-edit/src/modules/inputEditTable/sagas.js

```diff
@@ -23,23 +25,37 @@ export function* initialize() {
   ])
   yield put(actions.setEditForm({inputEditForm: editForm.body}))
   yield put(actions.setDataForm({inputDataForm: dataForm}))
-  yield call(loadData)
+  yield call(loadData, {})
 }
 
-export function* sortData({payload}) {
-  yield call(loadData, payload.sorting)
+export function* sortData({payload: {sorting}}) {
+  yield call(loadData, {newSorting: sorting})
 }
 
-export function* loadData(action, newSorting) {
+export function* loadData({newSorting, newSearchQueries, newPage}) {
   const {inputEntityKey} = yield select(inputSelector)
-  const sorting = newSorting || (yield select(inputEditSelector)).sorting
-  const data = yield call(rest.requestSaga, `inputEdit/${inputEntityKey}/data/search`, {
+  const dataForm = (yield select(inputEditTableSelector)).inputDataForm
+  const sorting = newSorting || (yield select(inputEditTableSelector)).sorting
+  const searchQueries = newSearchQueries || (yield select(searchQueriesSelector))
+  const currentPage = newPage || (yield select(inputEditPaginationSelector)).currentPage
+  const recordsPerPage = (yield select(inputEditPaginationSelector)).recordsPerPage
```

Comment:
you can use object destructuring: 
`const {recordsPerPage} = yield select(inputEditPaginationSelector)`